src/Sarif.Converters/HdfConverter.cs

```diff
@@ -126,7 +126,7 @@ private static (ReportingDescriptor, IList<Result>) SarifRuleAndResultFromHdfCon
                     RuleId = execJsonControl.Id,
                     Message = new Message
                     {
-                        Text = AppendPeriod(controlResult.CodeDesc),
+                        Text = AppendPeriod(string.IsNullOrWhiteSpace(controlResult.CodeDesc) ? execJsonControl.Desc : controlResult.CodeDesc),
```

Comment:
>IsNullOrWhiteSpace [](http://example.com/codeflow?start=51&length=18)

are you sure this logic is what you want? I think you have the arguments reversed maybe?

Reply:
Um, yes. :) Haven't had my coffee yet. The good news is my mistake squeezed a release note improvement out of you. :)